src/krb5/_set_password.pyx

```diff
@@ -33,14 +34,59 @@ cdef extern from "python_krb5.h":
         krb5_data *result_string
     ) nogil
 
-SetPasswordResult = collections.namedtuple(
-    'SetPasswordResult',
-    [
-        'result_code',
-        'result_code_string',
-        'result_string',
-    ],
-)
+class ADPolicyInfoProp(enum.IntFlag):
+    COMPLEX = 0x00000001
+    NO_ANON_CHANGEv = 0x00000002
+    NO_CLEAR_CHANGE = 0x00000004
+    LOCKOUT_ADMINS = 0x00000008
+    STORE_CLEARTEXT = 0x00000010
+    REFUSE_CHANGE = 0x00000020
+
+class ADPolicyInfo(typing.NamedTuple):
```

Comment:
As we aren't dealing with any C code here and this is pure Python it would make more sense to split this and the `ADPolicyInfoPro` enum into it's own Python file. This way you don't need to duplicate the definition in both the pyx and pyi files.

Reply:
done